GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/VertexNormal.java

```diff
@@ -0,0 +1,21 @@
+package org.gearvrf.x3d;
+
+public class VertexNormal {
+	public float[] vector = { 0, 0, 1 };
+
+	
+	public VertexNormal (float x, float y, float z) {
```

Comment:
Similar issue to Vertex.java.  Hope is to do away with this class altogether.  Just in the interest of getting a quick initial release, it is still there.